_drafts/SC1-3-5-autocomplete-valid.md

```diff
@@ -0,0 +1,177 @@
+
+- The element is **not** [[visible on the page]] or [[exposed to assistive technologies]]
+- The element is an `input` element a `type` of `hidden`, `button`, `submit` or `reset`
+- The element has a `disabled` or `aria-disabled="true"` attribute
+- The element has `tabindex="-1"` and has a [[semantic role]] that is not a [widget](https://www.w3.org/TR/wai-aria-1.1/#widget_roles)
+
+### Expectation 1
+
+The `autocomplete` attribute is a single term, or a space seperated list of terms.
+
+### Expectation 2
+
+The autocomplete term(s) follow the [HTML 5.2 specification](https://www.w3.org/TR/html52/sec-forms.html#autofill-detail-tokens), which requires that it/they match the following:
+
+1. Has a section-* value _(optional)_
+2. Has either "shipping" or "billing" _(optional)_
+3. Has "home", "work", "mobile", "fax" or "pager" _(optional, only for "email", "impp", "tel" or "tel-*")_
+4. Has a [correct autocomplete field][#correct-autocomplete-field] _(required)_
+
+**Note**: Autocomplete terms are case insensitive. When multiple terms are used, they must be used in the correct order.
+
+### Exepctation 3
+
+The [correct autocomplete field][#correct-autocomplete-field] is an [appropriate field for the form control][#appropriate-field-for-the-form-control].
+
+## Assumptions
+
+For this rule, it is assumed that the `autocomplete` attribute is not used on form fields that does not correspond to a autocomplete field described in the HTML 5.2 specification. If the `autocomplete` field is used to describe "custom" taxonomy, rather than that described in the specification, this rule may produce incorrect results.
+
+## Accessibility support
+
+While `autocomplete` in a promising technique for supporting personalisation in HTML, support for this is fairly limited. 
+
+## Background
+
+The intent of this rule is to ensure that the `autocomplete` attribute can be used to suport personalization. Many users may find it easier to fill out forms if those can be styled or layed out in a way that is familiar to them. Assistive technologies can do this when a form control is marked up in such a way that its purpose can be understood. For instance, assistive technologies could add familiar icons and colors to different fields, making it easier for the user to understand what the form does. 
+
+- https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html
+- https://www.w3.org/TR/WCAG21/#dfn-programmatically-determinable
+- https://www.w3.org/TR/html52/sec-forms.html#sec-autofill
+
+## Test Cases
+
+### Passed
+
+```html
+<input autocomplete="username" />
+```
+
+```html
+<select autocomplete="bday-month">
+  <option>January</option>
+  <option>...</option>
+</select>
+```
+
+```html
+<textarea autocomplete="Street-Address"></textarea>
+```
+
+```html
+<input autocomplete="Work EMail" />
+```
+
+```html
+<input autocomplete="section-partner email" />
+```
+
+```html
+<input autocomplete="section-primary shipping work email" />
```

Comment:
It would be good to add  a passing 3-component example like

`<input type="text" autocomplete="section-primary billing street-address />`